examples/solids/README.rst

```diff
@@ -42,7 +42,8 @@ The elasticity min-app is controlled via command-line options, the following of
      - `Poisson's ratio <https://en.wikipedia.org/wiki/Poisson%27s_ratio>`_, :math:`\nu < 0.5`
 
    * - :code:`-bc_clamp [int list]`
-     - List of face sets on which to displace by :code:`-bc_clamp_[facenumber]_translate [x,y,z]` and/or :code:`bc_clamp_[facenumber]_rotate [rx,ry,rz,theta]`
+     - List of face sets on which to displace by :code:`-bc_clamp_[facenumber]_translate [x,y,z]` and/or :code:`bc_clamp_[facenumber]_rotate [rx,ry,rz, c_0, c_1]`
```

Comment:
```suggestion
     - List of face sets on which to displace by :code:`-bc_clamp_[facenumber]_translate [x,y,z]` and/or :code:`bc_clamp_[facenumber]_rotate [rx,ry,rz,c_0,c_1]`
```

Reply:
I guess that was the style issue, extra spacing!

Reply:
Please apply this suggestion (click "Commit suggestion") and make sure you don't add trailing whitespace. You can instruct your editor to avoid it.